.travis.yml

```diff
@@ -30,7 +30,6 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
   allow_failures:
-    - php: nightly
```

Comment:
it looks like on master php7 is already passing https://travis-ci.org/symfony-cmf/Testing/builds/68312492